src/resources/views/edit.blade.php

```diff
@@ -1,76 +1,40 @@
+  @if(view()->exists('vendor.backpack.crud.form_content'))
+    @include('vendor.backpack.crud.form_content', ['fields' => $fields, 'entry' => $entry])
+  @else
+    @include('crud::form_content', ['fields' => $fields, 'entry' => $entry])
+  @endif
+{!! Form::close() !!}
 
-		  {!! Form::open(array('url' => $crud->route.'/'.$entry->getKey(), 'method' => 'put', 'files'=>$crud->hasUploadFields('update', $entry->getKey()))) !!}
-		  <div class="box">
-		    <div class="box-header with-border">
-		    	@if ($crud->model->translationEnabled())
-			    	<!-- Single button -->
-					<div class="btn-group pull-right">
-					  <button type="button" class="btn btn-sm btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-					    Language: {{ $crud->model->getAvailableLocales()[$crud->request->input('locale')?$crud->request->input('locale'):App::getLocale()] }} <span class="caret"></span>
-					  </button>
-					  <ul class="dropdown-menu">
-					  	@foreach ($crud->model->getAvailableLocales() as $key => $locale)
-						  	<li><a href="{{ url($crud->route.'/'.$entry->getKey().'/edit') }}?locale={{ $key }}">{{ $locale }}</a></li>
-					  	@endforeach
-					  </ul>
-					</div>
-					<h3 class="box-title" style="line-height: 30px;">{{ trans('backpack::crud.edit') }}</h3>
-				@else
-					<h3 class="box-title">{{ trans('backpack::crud.edit') }}</h3>
-				@endif
-		    </div>
-		    <div class="box-body row">
-		      <!-- load the view from the application if it exists, otherwise load the one in the package -->
-		      @if(view()->exists('vendor.backpack.crud.form_content'))
-		      	@include('vendor.backpack.crud.form_content')
-		      @else
-		      	@include('crud::form_content', ['fields' => $fields])
-		      @endif
-		    </div><!-- /.box-body -->
-
-            <div class="box-footer">
-
-                @include('crud::inc.form_save_buttons')
-
-		    </div><!-- /.box-footer-->
-		  </div><!-- /.box -->
-		  {!! Form::close() !!}
-	</div>
-</div>
 @endsection
 
 @section('after_styles')
-	<link rel="stylesheet" href="{{ asset('vendor/backpack/crud/css/crud.css') }}">
-	<link rel="stylesheet" href="{{ asset('vendor/backpack/crud/css/form.css') }}">
-	<link rel="stylesheet" href="{{ asset('vendor/backpack/crud/css/edit.css') }}">
+  @stack('crud_fields_styles')
+  <link rel="stylesheet" href="{{ asset('vendor/backpack/crud/css/crud.css') }}">
+  <link rel="stylesheet" href="{{ asset('vendor/backpack/crud/css/form.css') }}">
+  <link rel="stylesheet" href="{{ asset('vendor/backpack/crud/css/edit.css') }}">
 @endsection
 
 @section('after_scripts')
-	<script src="{{ asset('vendor/backpack/crud/js/crud.js') }}"></script>
-	<script src="{{ asset('vendor/backpack/crud/js/form.js') }}"></script>
-	<script src="{{ asset('vendor/backpack/crud/js/edit.js') }}"></script>
-@endsection
\ No newline at end of file
+  @stack('crud_fields_scripts')
```

Comment:
I don't know if @tabacitu enforces any particular type of indentation however inconsistent indentation is one of my own pet annoyances (I'm sure we have them).

It's not important enough to block a PR, I guess.